src/System.Security.Cryptography.Xml/tests/XmlDsigXsltTransformTest.cs

```diff
@@ -182,8 +182,11 @@ public void OnlyInner()
             doc.LoadXml(test);
 
             transform.LoadInnerXml(doc.ChildNodes);
-            Stream s = (Stream)transform.GetOutput();
-            Assert.Throws<ArgumentNullException>(() => Stream2Array(s));
+            Assert.Throws<ArgumentNullException>(() =>
```

Comment:
@krwq I appreciate that.  But my point is that if you're making a change to the test you should make the right change.  It seems weird to me to have a two line lambda expecting an ArgumentNullException.  If the problem is that Mono and NetFx generate them from different places, add a comment.  Otherwise, make the test test what we want it to test.... whatever that may be.